frontend/src/metabase/dashboard/actions/cards.js

```diff
@@ -213,3 +229,58 @@ export const addActionToDashboard =
       }),
     );
   };
+
+export const autoApplyParametersToNewCard =
```

Comment:
I think this might be able to be split out as well, but considering the lack of use-cases in other parts of the code I don't think it's super necessary.